open_humans/templates/account/signup-form.html

```diff
@@ -7,7 +7,7 @@
       action="{% url 'email_signup' %}" id="signup-form">
   {% csrf_token %}
 
-  <input type="hidden" name="next" value="{% get_next_url %}">
+  <input type="hidden" name="next" value="{{ request.path }}">
```

Comment:
I guess this is unneeded now?

Reply:
Actually keep, same as above comment. The value needs to change though, to match the next parameter...